k8s/perftest/common/rpe/rpe-service-auth-provider.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: rpe-service-auth-provider
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-service-auth-provider
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "ed3c45ae-01e1-4263-b57f-178a1ba004be"
+      image: hmctspublic.azurecr.io/hmcts/rpe-service-auth-provider:prod-9b79e227
+      ingressHost: rpe-service-auth-provider.service.core-compute-aat.internal
```

Comment:
this looks wrong

Reply:
Please do not leave comments like the above :-)
Rest instead!

Reply:
sure..